pkg/arvo/mar/group/initial.hoon

```diff
@@ -1,14 +1,13 @@
-/+  *group-json
```

Comment:
This file is no longer necessary. Please incorporate `%initial` into the %group-update.

Reply:
Sorry for duplicating the comment! Weird internet